magento/magento2
# Fixed issue 18794 - My billing and shipping address are the same" checkbox won't be unchecked when multiple payment methods are available.

### Description (*)

My billing and shipping address are the same" checkbox won't be unchecked when multiple payment methods are available.

### Fixed Issues (if relevant)

1. magento/magento2#18794: My billing and shipping address are the same" checkbox won't be unchecked when multiple payment methods are available.

### Manual testing scenarios (*)

1.Add to cart a product, and go to checkout.
2.Select any address, and click "Next" button.
3.Select any payment method, and check "My billing and shipping address are the same
" checkbox.
4.Return to Shipping step.
5.Select other address, and click "Next" button. 

### Contribution checklist (*)
